src/config/dark-sites.config

```diff
@@ -91,6 +91,7 @@ cascadr.co
 cdnnow.pro
 cheat.sh
 checkra.in
+chess.com
```

Comment:
https://www.chess.com/register
My eyes, this isn't dark.

Reply:
Can I exclude a sub website?

Reply:
Nope

Reply:
Ok thanks, I'll remove chess.com then